lib/SQL/Translator/Producer/PostgreSQL.pm

```diff
@@ -236,6 +246,25 @@ sub produce {
   my @output;
   push @output, header_comment unless ($no_comments);
 
+  for my $sequence ($schema->get_sequences) {
+
+  my (@sequence_defs, @fks);
+    my ($sequence_def, $fks) = create_sequence(
+      $sequence,
+      {
+        generator        => $generator,
+        no_comments      => $no_comments,
+        postgres_version => $postgres_version,
+        # add_drop_sequence   => $add_drop_sequence,
+        # type_defs        => \%type_defs,
```

Comment:
what's with the dead comments here? also what's supposed to controll the `add_drop_sequence`?

Reply:
Removed.